05-investment-calculator/src/components/Results.jsx

```diff
@@ -0,0 +1,42 @@
+import { calculateInvestmentResults, formatter } from "../util/investment";
+
+export default function Results({ input }) {
+  const resultsData = calculateInvestmentResults(input);
+  const initialInvestment =
+    resultsData[0].valueEndOfYear -
+    resultsData[0].interest -
+    resultsData[0].annualInvestment;
+
+  return (
+    <table id="result">
```

Comment:
Try to avoid using id because if you using the same component twice the id will conflict